etc/behat/services/contexts.xml

```diff
@@ -53,6 +53,7 @@
         <parameter key="sylius.behat.context.ui.paypal.class">Sylius\Behat\Context\Ui\PaypalContext</parameter>
         <parameter key="sylius.behat.context.ui.customer.class">Sylius\Behat\Context\Ui\CustomerContext</parameter>
         <parameter key="sylius.behat.context.ui.theme.class">Sylius\Behat\Context\Ui\ThemeContext</parameter>
+        <parameter key="sylius.behat.context.ui.admin.managing_country.class">Sylius\Behat\Context\Ui\Admin\ManagingCountryContext</parameter>
```

Comment:
@lchrusciel and @pjedrzejewski discussed about it [here](https://github.com/Sylius/Sylius/pull/4395#discussion_r55678274)